tests/checkpoint/test_simple_checkpoint.py

```diff
@@ -920,6 +920,10 @@ def runtime_batch_request():
         datasource_name="my_datasource",
         data_connector_name="my_runtime_data_connector",
         data_asset_name="users",
+        batch_identifiers={"pipeline_stage_name": "first"},  # defined in fixture
+        runtime_parameters={
+            "query": "SELECT * FROM taxi_data"
+        },  # not actually run, but used to test configuration
```

Comment:
I think we generally wouldn't test for something like that but since we've changed the public API in some measurable way, it doesn't hurt?

I'm okay either way I'm not sure what the approach is here.